frontend/src/pages/DegreeWizard/steps/DegreeStep.jsx

```diff
@@ -1,26 +1,25 @@
 import React, { useEffect } from "react";
-// import axios from 'axios';
-import { Menu, Typography, Button } from "antd";
+import axios from "axios";
+import { Menu, Typography, Button, Input } from "antd";
 import { degreeActions } from "../../../actions/degreeActions";
 import { Link } from "react-scroll";
 import { useDispatch, useSelector } from "react-redux";
 import "./steps.less";
+import { useSpring, animated } from "react-spring";
+import { springProps } from "../spring";
 
 const { Title } = Typography;
-export const DegreeStep = ({ isYearsSet }) => {
+export const DegreeStep = ({ incrementStep, currStep }) => {
   const dispatch = useDispatch();
   const programCode = useSelector((store) => store.degree.programCode);
   const [input, setInput] = React.useState("");
   const [options, setOptions] = React.useState(null);
 
   const fetchAllDegrees = async () => {
-    // const res = await axios.get("http://localhost:8000/api/getPrograms");
+    // const res = await axios.get("http://localhost:8000/programs/getPrograms");
     // setOptions(res.data["programs"]);
     setOptions({
-      3778: "Bachelor Computer Science",
-      3779: "Bachelor of Science",
-      3777: "Bachelor of Arts",
-      3775: "Bachelor of Commerce/Law",
+      3778: "Computer Science",
```

Comment:
Hmm i think I should do this on the backend - this restriction is something that the backend needs to deal with, not the frontend. Ill change the endpoint to just return CS for now.